scripts/exregional_run_prepstart.sh

```diff
@@ -464,13 +469,15 @@ fi
 #  surface cycling
 #
 #-----------------------------------------------------------------------
+#SFC_CYC=2
+if_update_ice="TRUE"
```

Comment:
Do we have an parameter in config.sh to turn on/off if_update_ic?

Reply:
Yes, We have. No true/false but use  cycle time to turn it on. If you set 00-23, it will be on with those cycles. The off is 99.

Reply:
Got it. But here line 473 hardcodes "if_update_ice" to true. Is this temporarily?

Reply:
I did not decide if this will be an option or remove it. Right now, cycle surface and update_ice go together.